src/ui/user/games/gamesView.tsx

```diff
@@ -54,7 +55,7 @@ function onTap(ctrl: IUserGamesCtrl, e: Event) {
         const userColor: Color = g.players.white.userId === userId ? 'white' : 'black'
         positionsCache.set(g.id, { fen: g.fen, orientation: userColor })
         const mePlaying = session.getUserId() === userId
-        if (mePlaying) {
+        if (mePlaying || (g.source !== 'import' && g.status.id < gameStatus.ids.aborted)) {
```

Comment:
You should use this function instead: 

https://github.com/veloce/lichobile/blob/master/src/lichess/game.ts#L29

which does the exact same check

Reply:
ok, nevermind my comment, this game object is not compatible...